snark-keys/src/header.rs

```diff
@@ -45,12 +45,12 @@ pub enum KeyIdentifier {
     TransactionSnark,
 }
 
-#[derive(Debug, Deserialize)]
+#[derive(Debug, Default, Deserialize)]
```

Comment:
Hopefully we can remove these changes if we can drop the dependency

Reply:
Fixed